### PR TITLE
cirow/token expiration support

### PR DESCRIPTION
**Pull request by cirow**
Greetings!
Thank you for sharing this, this repo helped me a lot in my day job. We've needed the tokens to expire though, so I've made some changes to add token and refresh token expiration support.
Let me know if you have any questions/concerns.

### CODE REVIEW

File: input.html [hunk 1]
```diff
@@ -35,6 +35,10 @@ <h2>Create token & grant access</h2>
                     <label for="expires_in">Token expiry (in seconds): </label>
                     <input type="text" class="form-control" value="3600" name="expires_in" id="expires_in"/>
                 </div>
+                <div class="form-group">
+                    <label for="refresh_token_expires_in">Refresh Token expiry (in seconds): </label>
+                    <input type="text" class="form-control" value="7200" name="refresh_token_expires_in" id="refresh_token_expires_in"/>
```
Comment (erosb) [inline input.html]: Hello, can this be `<input type="number" />` please?

Comment (cirow) [reply input.html]: Sure, should I change the token expires_in too? Or should I just do it to the refresh_token_expires_in?

Comment (erosb) [reply input.html]: Sure, that makes sense, thanks!

Comment (cirow) [reply input.html]: Done!

**Comment by erosb** [comment]
Thank you for your contribution @cirow !

@matevarga can you please merge this PR? Thanks.

**Pull request merged**